src/assets/scripts/aircraft/AircraftInstanceModel.js

```diff
@@ -1705,6 +1705,42 @@ export default class Aircraft {
     }
 
     /**
+     * Checks if the aircraft is inside the airspace of a specified airport
+     *
+     * @for AircraftInstanceModel
+     * @method isInsideAirspace
+     * @param  {airport} airport the airport whose airspace we are checking
+     * @return {Boolean}
+     * @private
+     */
+    isInsideAirspace(airport) {
+        let withinAirspaceLateralBoundaries = this.distance <= airport.ctr_radius;
+        const withinAirspaceAltitudeRange = this.altitude <= airport.ctr_ceiling;
+
+        if (!_isNil(airport.perimeter)) {    // polygonal airspace boundary
+            withinAirspaceLateralBoundaries = point_in_area(this.position, airport.perimeter);
+        }
+
+        return withinAirspaceAltitudeRange && withinAirspaceLateralBoundaries;
+    }
+
+    /**
+     * Aircraft has "weight-on-wheels" (on the ground)
+     * @for AircraftInstanceModel
+     * @method isOnGround
+     */
+    isOnGround() {
+        const error_allowance_ft = 5;
+        const apt = window.airportController.airport_get();
```

Comment:
could you rename this `airport`?  This is sort of a nit-pick, I know.  I much prefer these things to be spelled out instead of shortened like this.  Its just clearer about what, exactly, that var represents.